force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -347,22 +397,21 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static List<String> getHighPaidEmployees() {
         // Initialize a Map
-        Map<String, Integer> employeeSalaries = new Map<String, Integer>();
-
-        // Add key-value pairs to the Map
-        // add John Doe => 50000
-        // add Jane Smith => 60000
-        // add Sam Brown => 55000
-        // add Alice Johnson => 65000
+        Map<String, Integer> empSalaries = new Map<String, Integer>();
 
-        // Initialize a list to store the names of high paid employees
+        empSalaries.put('John Doe', 50000); // add John Doe => 50000
+        empSalaries.put('Jane Smith', 60000);// add Jane Smith => 60000
+        empSalaries.put('Sam Brown', 55000);// add Sam Brown => 55000
+        empSalaries.put('Alice Johnson', 65000);// add Alice Johnson => 65000
```

Comment:
Try using a static variable for employeeSalaries for question 19 and 20 to reduce duplicates since the values are the same